elephant/test/test_spade.py

```diff
@@ -348,6 +348,41 @@ def test_fpgrowth_fca(self):
         assert_array_equal(sorted(mining_results_ffca[0][1]), sorted(
             mining_results_fpg[0][1]))
 
+    def test_spade_output(self):
+
+        np.random.seed(0)
+
+        n_spiketrains = 3
+
+        poisson_process = stg.StationaryPoissonProcess(
+            rate=20 * pq.Hz, t_stop=5 * pq.s)
+        spiketrains = poisson_process.generate_n_spiketrains(
+            n_spiketrains=n_spiketrains)
+        bin_size = 5 * pq.ms
+
+        spade_output = spade.spade(
+            spiketrains,
+            bin_size=bin_size,
+
+            # +1 to pattern length to be able to find it again
+            winlen=2,
+            min_spikes=n_spiketrains,
+            max_spikes=n_spiketrains,
```

Comment:
Is the usage of `n_spiketrains` in this context specifically due to its value being 3? Or does the number of spiketrains have a direct impact on `min_spikes` and `max_spikes`?

Reply:
I chose arbitrarily the 3 here. I just wanted to have a case, where I can have an unwanted autopatterns, that is with a repeating neuron. - This I get with this Poisson spike trains and choice of parameters.